### ISSUE TITLE
Sent $20 in total and sent email no pro unlocked

### ISSUE THREAD

**Issue by tomjb826** [croccio/CarTube#82]
Been waiting for pro to unlock for about 5 days now and still hasn't been unlocked here's my ID 97b9e4a3-6e69-4798-9ba8-c21eb13f6429-LM-V600

Thanks for any help.

**Comment by schumi2004** [comment]
I would ditch this repo/solution. I also asked a question about licensing and no answer received, unfortunately you paid already.

I would go for WebViewAuto, at first it looks like a browser but you can open and play YouTube in it same as with CarTube without root and as a bonus your getting a browser along with it.

The repo (will add link later) doesn't have recent apk available but you can download the one i compiled or compile it your self, its not that hard.

https://www.dropbox.com/s/rglw9jk2y474u7g/WebViewAuto-activity-debug%20v1.0.apk?dl=0

**Comment by tomjb826** [comment]
Thank you so much I really appreciate it

**Comment by schumi2004** [comment]
Bookmark it quickly before my comments get deleted ;)

**Comment by costa802** [comment]
Ciao, ho scaricato l'app webviewauto però non c'è su android auto, come devo fare?

**Comment by tomjb826** [comment]
Yeah actually I have the same issue just tried it not showing up hmmm

**Comment by costa802** [comment]
Speriamo che schumi2004 ci può dire come fare!

**Comment by tomjb826** [comment]
I'm sure a google search can help us sir I'll give it a shot tomorrow :)

**Comment by schumi2004** [comment]
Did you guys enabled Developer options and checked Unknown sources?
Enable Developer settings in Android Auto
Enable 'Unknown sources' checkbox

That's how i did it.

**Comment by tomjb826** [comment]
Yes sir that's exactly how I did it maybe Android auto been updated since and it's been patched?
